### DISCUSSION TITLE
New design for version 6.0 - boards available?

### DISCUSSION THREAD

**Discussion by jfseaman** [gusmanb/logicanalyzer#154] [General]
I am very interested if you have new board design boards available.

**Comment by gusmanb** [comment]
> I am very interested if you have new board design boards available.

All the boards of the new batch is already reserved, send me an email to logicanalyzerorders@gmail.com and when a new one is ready I will send you an email.

**Reply by Wireb** [reply]
Is the design files for the V6 PCBs available for those that want to build their own?
